src/QMCWaveFunctions/einspline_helper.hpp

```diff
@@ -243,6 +243,37 @@ inline void fix_phase_rotate_c2c(const Array<std::complex<T>, 3>& in,
     }
 }
 
+template<typename T, typename T1, typename T2>
+inline void fix_phase_norotate_c2c(const Array<std::complex<T>, 3>& in,
```

Comment:
This function should be simplified to only copy from `in` to `out_r` and `out_i`.
Setting phase_r and phase_i can be set at the caller side directly.

Reply:
I'm OK having an auxiliary function here, like "real_component_split_c2c" that 1.) performs a direct copy for the real and imaginary parts as Ye suggests, and 2.) sets the phase phi_r=1 and phi_i=0.  I'd rather have the phase determined by the choice of operation than leave this to the user.  Not necessary though.

Reply:
done